docker/ludwig-ray/Dockerfile

```diff
@@ -9,6 +9,8 @@
 #   model serving
 #
 
+FROM python:3.8.12-slim
```

Comment:
Should make this match with the ray gpu image, so something like `rayproject/ray:1.9.2-py38` probably.

Reply:
this can be deleted, since we have `FROM rayproject/ray:latest` below right?